Source/Modules/go.cxx

```diff
@@ -10,6 +10,7 @@
 #include "swigmod.h"
 #include "cparse.h"
 #include <ctype.h>
+#include <stdint.h>
```

Comment:
I really don't know what the SWIG portability rules are.  Can you ask on swig-devel whether it's OK to #include <stdint.h>?  I don't see any current SWIG code that does.  I also don't see any uses of int64 or "long long" outside of tests.